eng/scripts/Language-Settings.ps1

```diff
@@ -87,15 +87,21 @@ function Publish-javascript-GithubIODocs ($DocLocation, $PublicArtifactLocation)
   $PublishedDocs = Get-ChildItem "$($DocLocation)/documentation" | Where-Object -FilterScript { $_.Name.EndsWith(".zip") }
 
   foreach ($Item in $PublishedDocs) 
-  {
-    $PkgName = "azure-$($Item.BaseName)"
-    Write-Host $PkgName
+  {    
     Expand-Archive -Force -Path "$($DocLocation)/documentation/$($Item.Name)" -DestinationPath "$($DocLocation)/documentation/$($Item.BaseName)"
     $dirList = Get-ChildItem "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)" -Attributes Directory
 
     if ($dirList.Length -eq 1)
     {
       $DocVersion = $dirList[0].Name
+      $pkgs = Get-ChildItem -Path $PublicArtifactLocation -Include "*.tgz" -Recurse -File
```

Comment:
Can you please verify that this will only ever have one tgz? I believe that to be true currently based on https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/stages/archetype-js-release.yml#L158-L162 but it would be good to confirm.

Reply:
Yes, This should only be one package that is supposed to be released in that job. Copy steps should find only one package. But I will add this as exact check to be on safer side.